server/src/main/java/com/aws/greengrass/cli/CLIService.java

```diff
@@ -221,32 +225,28 @@ private synchronized void generateCliIpcInfo() throws UnauthenticatedException,
         Path authTokenDir = kernel.getNucleusPaths().cliIpcInfoPath();
         revokeOutdatedAuthTokens(authTokenDir);
 
-        // [P41372857]: Support Windows group permissions
-        if (Exec.isWindows) {
+        Topic authorizedGroups = config.find(CONFIGURATION_CONFIG_KEY,
+                Exec.isWindows ? AUTHORIZED_WINDOWS_GROUPS : AUTHORIZED_POSIX_GROUPS);
+        if (authorizedGroups == null) {
```

Comment:
use Coerce.toString() before checking for null. The Topic may not be null, but the value of that topic is null